spec/namespaces/ml.yaml

```diff
@@ -130,36 +130,52 @@ components:
     ml.search_models:
       content:
         application/json:
-          $ref: '../schemas/ml._common.yaml#/components/schemas/SearchModelsQuery'
+          schema:
+            $ref: '../schemas/ml._common.yaml#/components/schemas/SearchModelsQuery'
   responses:
     ml.register_model_group@200:
       content:
         application/json:
-          $ref: '../schemas/ml._common.yaml#/components/schemas/ModelGroupRegistration'
+          schema:
+            $ref: '../schemas/ml._common.yaml#/components/schemas/ModelGroupRegistration'
     ml.get_model_group@200:
       content:
         application/json:
-          $ref: '../schemas/ml._common.yaml#/components/schemas/ModelGroup'
+          schema:
+            $ref: '../schemas/ml._common.yaml#/components/schemas/ModelGroup'
     ml.delete_model_group@200:
       content:
         application/json:
-          $ref: '../schemas/ml._common.yaml#/components/schemas/ModelGroup'
+          schema:
+            $ref: '../schemas/_common.yaml#/components/schemas/WriteResponseBase'
     ml.register_model@200:
       content:
         application/json:
-          $ref: '../schemas/ml._common.yaml#/components/schemas/Task'
+          schema:
```

Comment:
Is this not a task? In tests we register a model then wait it the task to complete.

Reply:
The response is not the entire Task object, which is different from `get_task`'s response. This only returns the task_id for you to check on it later.